_blogposts/2020-02-09-release-9-0.mdx

```diff
@@ -0,0 +1,134 @@
+---
+author: hongbo
+date: "2020-02-09" 
+previewImg:
+category: compiler
+title: ReScript 9.0
+description: |
+---
+
+## Introduction
+
+ReScript is a soundly typed language with an optimizing compiler focused on the JS platform. 
```

Comment:
It's hard to review with the line breaks in each paragraph, can you remove the line breaks, so that every paragraph is soft-wrapped on one line?

Reply:
I am not sure if I follow you, you mean no new line for each paragraph?